packages/react-components/react-nav-preview/stories/NavDrawerInline/index.stories.tsx

```diff
@@ -0,0 +1,8 @@
+import { NavDrawer } from '@fluentui/react-nav-preview';
+
+export { NavDrawerInline } from './NavDrawerInline.stories';
+
+export default {
+  title: 'Preview Components/NavDrawerInline',
+  component: NavDrawer,
+};
```

Comment:
Imo, both the stories from Overlay vs Inline are variations of NavDrawer, so they should live in a NavDrawer story like this:
- stories
  - NavDrawer
    - index.stories.tsx
    - NavDrawerOverlay.stories.tsx
    - NavDrawerInline.stories.tsx

This is because when it's moved out of preview, you'll be taking up 4 component spaces in the docs. Ideally all Nav related stories should live under one place. Alternatively, you could do something like Button and have
- Nav
  - Nav
  - NavDrawer

Reply:
Agreed! I'll rejigger the story structure next week when I start adding it to the main story book.